src/puppetlabs/jdbc_util/core.clj

```diff
@@ -190,14 +190,21 @@
       :pg_get_serial_sequence))
 
 (defn reconcile-sequence-for-column!
-  "Finds the sequence associated with the given column and sets it to the
-  current maximum value in that column, so that the next value in the sequence
-  will be the current maximum + 1 (or 0, if the table is empty). If the column
-  has no associated sequence, throws an Exception."
+  "Finds the sequence associated with the given column and sets it
+  to the greatest of:
+    * the maximum value currently in the column + 1
+    * the current value of the sequence
+    * 1 (if the column is empty)
```

Comment:
It defaults to 1 and `is_called` is false, so that the first call is special cased. :|